albatross/models/least_squares.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2018 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef ALBATROSS_MODELS_LEAST_SQUARES_H
+#define ALBATROSS_MODELS_LEAST_SQUARES_H
+
+/*
+ * Here we define a LeastSquaresRegression model which is less because
```

Comment:
less...?

Reply:
that was just generally a confusing out of date comment, I removed it entirely.